### PR TITLE
Tensorflow: fix #77626

### LINKED ISSUE #77626: python3Packages.tensorflow: undefined symbols in python3.7/site-packages/tensorflow_core/contrib/tensor_forest/python/ops/_model_ops.so
**Describe the bug**

Tensorflow 1.15.0 on current unstable commit (e.g. 59c4035e58e) fails to load the shared library `lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/python/ops/_model_ops.so`. It tries to find the symbol `_ZNK10tensorflow12tensorforest20DecisionTreeResource12TraverseTreeERKSt10unique_ptrINS0_13TensorDataSetESt14default_deleteIS3_EEiPiPNS0_8TreePathE`, fails and bails out.

The missing symbol demangles to: `tensorflow::tensorforest::DecisionTreeResource::TraverseTree(std::unique_ptr<tensorflow::tensorforest::TensorDataSet, std::default_delete<tensorflow::tensorforest::TensorDataSet> > const&, int, int*, tensorflow::tensorforest::TreePath*) const`

**To Reproduce**
`nix-shell -I nixpkgs=. -E 'with import <nixpkgs> {}; (python3.withPackages (p: [p.tensorflow])).env' --run "python -c 'from tensorflow.contrib import tensor_forest'"`

```bash
$ nix-shell -I nixpkgs=. -E 'with import <nixpkgs> {}; (python3.withPackages (p: [p.tensorflow])).env' --run "python -c 'from tensorflow.contrib import tensor_forest'"
WARNING:root:Limited tf.compat.v2.summary API due to missing TensorBoard installation.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<frozen importlib._bootstrap>", line 1019, in _handle_fromlist
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow/__init__.py", line 50, in __getattr__
    module = self._load()
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow/__init__.py", line 44, in _load
    module = _importlib.import_module(self.__name__)
  File "/nix/store/bzvpdbzdy3a1qba8f9ynrzyz811c10pp-python3-3.7.6/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/__init__.py", line 90, in <module>
    from tensorflow.contrib import tensor_forest
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/__init__.py", line 21, in <module>
    from tensorflow.contrib.tensor_forest.client import *
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/__init__.py", line 21, in <module>
    from tensorflow.contrib.tensor_forest.client import *
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/client/__init__.py", line 22, in <module>
    from tensorflow.contrib.tensor_forest.client import random_forest
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/client/random_forest.py", line 27, in <module>
    from tensorflow.contrib.tensor_forest.python import tensor_forest
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/python/__init__.py", line 21, in <module>
    from tensorflow.contrib.tensor_forest.python import tensor_forest
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/python/tensor_forest.py", line 30, in <module>
    from tensorflow.contrib.tensor_forest.python.ops import model_ops
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/python/ops/model_ops.py", line 42, in <module>
    resource_loader.get_path_to_datafile("_model_ops.so"))
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/util/loader.py", line 56, in load_op_library
    ret = load_library.load_op_library(path)
  File "/nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/python/framework/load_library.py", line 61, in load_op_library
    lib_handle = py_tf.TF_LoadLibrary(library_filename)
tensorflow.python.framework.errors_impl.NotFoundError: /nix/store/nyx0n2hpw40n90izaxfv77qjqdpfrijk-python3-3.7.6-env/lib/python3.7/site-packages/tensorflow_core/contrib/tensor_forest/python/ops/_model_ops.so: undefined symbol: _ZNK10tensorflow12tensorforest20DecisionTreeResource12TraverseTreeERKSt10unique_ptrINS0_13TensorDataSetESt14default_deleteIS3_EEiPiPNS0_8TreePathE
```

**Expected behavior**
The import should succeed as it did with TF 1.14.0.

**Additional context**
I tried reproducing this with the official TF 1.15.0 docker image but failed so it seems like a unique nipxkgs issue.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  - tensorflow
  - python3Packages.tensorflow
```

cc @timokau @prusnak

### PR DESCRIPTION
**Pull request by andir**
###### Motivation for this change
This fixes #77626 by using an older Bazel version.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/build-managers/bazel/bazel_0/default.nix [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+in
+stdenv.mkDerivation rec {
+
+  version = "0.26.0";
+
+  meta = with lib; {
+    homepage = "https://github.com/bazelbuild/bazel/";
+    description = "Build tool that builds code quickly and reliably";
+    license = licenses.asl20;
+    maintainers = [ maintainers.mboes ];
```
Comment (FRidh) [inline pkgs/development/tools/build-managers/bazel/bazel_0/default.nix]: cc @mboes

**Comment by Profpatsch** [comment]
I thought we added back bazel 1 to fix tensorflow?

File: pkgs/development/tools/build-managers/bazel/bazel_0/default.nix (hunk 1 above)
Comment (Profpatsch) [reply pkgs/development/tools/build-managers/bazel/bazel_0/default.nix]: I guess tensorflow maintainers should add themselves here.

**Review by Profpatsch — COMMENTED** [review]
Looks like bazel 1 was not old enough after all. 
*shaking fist against the sky screaming* Tensorfloooooow

File: pkgs/development/tools/build-managers/bazel/bazel_0/default.nix (hunk 1 above)
Comment (andir) [reply pkgs/development/tools/build-managers/bazel/bazel_0/default.nix]: Fixed

**Comment by andir** [comment]
Removed the maintainers line. As I am not really a tensorflow maintainer just someone that actually tries to unbreak it (once) I didn't add myself there.

**Pull request merged**